test/cypress/utils/apple-pay-js-api-stubs.js

```diff
@@ -29,9 +29,17 @@ const getMockApplePayClass = (validPaymentRequestResponse, email = null) => {
       this._onvalidatemerchant = value
     }
 
+    get onvalidatemerchant () {
```

Comment:
Are these new functions?

Reply:
Standard 16 warns against getters without setters. 

We could switch this off I suppose, wdyt?